app/src/main/java/ip/team13/petowner/ui/login/LoginFragment.kt

```diff
@@ -1,13 +1,29 @@
 package ip.team13.petowner.ui.login
 
+import android.os.Bundle
+import android.view.View
+import androidx.navigation.fragment.findNavController
+import com.google.android.gms.common.SignInButton
 import ip.team13.petowner.R
-import ip.team13.petowner.core.BaseFragment
 import org.koin.androidx.viewmodel.ext.android.viewModel
 
-class LoginFragment : BaseFragment() {
+class LoginFragment : GoogleLoginFragment() {
 
     override val layout: Int
         get() = R.layout.login_screen
 
     override val viewModel: LoginViewModel by viewModel()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        view.findViewById<SignInButton>(R.id.sign_in_button)?.setOnClickListener {
```

Comment:
Can we use [Kotlin Synthetics](https://developer.android.com/topic/libraries/view-binding/migration) to access the views, instead of the `findViewById` call which feels a bit too Java style?

Reply:
Nvm. I meant [View Binding](https://developer.android.com/topic/libraries/view-binding) instead, but I will add support for it in the `BaseFragment` later. It's fine for now